### PR TITLE
fix(Scripts/Ulduar): Yogg-Saron portals should be interactable.

### LINKED ISSUE #11140: [ulduar] Yogg-Saron's can't be killed
### Current Behaviour

In phase P2, you cannot click 'descent into madness'(npc id 34072) to enter yogsaron's brain,put the mouse on it and there is no icon of gear

### Expected Blizzlike Behaviour

In phase P2, you can click 'descent into madness' to enter yogsaron's brain

It can be seen from 3:24
https://www.youtube.com/watch?v=odmXq4OJnx4

### Source

_No response_

### Steps to reproduce the problem

1.Enter P2 phase
2.click 'Descend Into Madness'

### Extra Notes

_No response_

### AC rev. hash/commit

86e438a1e7c508ef2ec844ce39535ae365748f57

### Operating system

ubuntu 20.04

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #11140

## Issues Addressed:

- Closes #11140

## Tests Performed:

- Not tested.

## How to Test the Changes:

1.Enter P2 phase
2.click 'Descend Into Madness'

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by UltraNix** [comment]
@mpfans Please, test it.

File: data/sql/updates/pending_db_world/rev_1648290893075572700.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1648290893075572700');
+
+UPDATE `creature_template` SET `npcflag`=16777216 WHERE `entry`=34072;
+DELETE FROM `npc_spellclick_spells` WHERE `npc_entry`=34072;
+INSERT INTO `npc_spellclick_spells` VALUES
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1648290893075572700.sql:5]:
```suggestion
INSERT INTO `npc_spellclick_spells` (`npc_entry`, `spell_id`, `cast_flags`, `user_type`) VALUES
```

**Review by Nefertumm — APPROVED** [review]

**Comment by mpfans** [comment]
testing,Need more test reports

**Comment by mpfans** [comment]
tested,work now

**Review by IntelligentQuantum — APPROVED** [review]

**Pull request merged**